RealEstate/apps/core/templates/core/report.html

```diff
@@ -1,8 +1,226 @@
+    var data1 = [{% for category, weight1, weight2 in categoryImportance %}{{weight1}}*100{% if not forloop.last %},{% endif %}{% endfor %}];
+    var data2 = [{% for category, weight1, weight2 in categoryImportance %}{{weight2}}*100{% if not forloop.last %},{% endif %}{% endfor %}];
+    var data = {
+        labels: catLabel,
+        datasets: [
+            {
+                label: "{{ homebuyer1 }}",
+                fillColor: "rgba(217, 95, 73, 0.9)",
+                strokeColor: "rgba(217, 95, 73, 0.8)",
+                highlightFill: "rgba(217, 95, 73, 0.8)",
+                highlightStroke: "rgba(217, 95, 73, 1)",
+                data: data1
+            },
+            {
+                label: "{{ homebuyer2 }}",
+                fillColor: "rgba(237, 194, 51, 0.9)",
+                strokeColor: "rgba(237, 194, 51, 0.8)",
+                highlightFill: "rgba(237, 194, 51, 0.8)",
+                highlightStroke: "rgba(237, 194, 51, 1)",
+                data: data2
+            }
+        ]
+    };
+    var ctx = document.getElementById("myChart").getContext("2d");
+    var myBarChart = new Chart(ctx).Bar(data, {
+        barShowStroke: false,
+        scaleShowGridLines: false,
+        barValueSpacing : 20,
+        showTooltips: false,
+    });
+    
+    var pie1 = [
+    {% for color, category, weight in pie1 %}
+        {
+            value : {{weight}},
+            color : '{{color}}',
+            label : '{{category}}'
+        }{% if not forloop.last %},{% endif %}
+    {% endfor %}
+    ]
+    
+    var pie2 = [
+    {% for color, category, weight in pie2 %}
+        {
+            value : {{weight}},
+            color : '{{color}}',
+            label : '{{category}}'
+        }{% if not forloop.last %},{% endif %}
+    {% endfor %}
+    ]
+    
+    var ctx1 = document.getElementById("PieChart1").getContext("2d");
+    var pieChart1 = new Chart(ctx1).Pie(pie1,{
+        barShowStroke: false,
+        scaleShowGridLines: false,
+        barValueSpacing : 20,
+    });
+    
+    var ctx2 = document.getElementById("PieChart2").getContext("2d");
+    var pieChart2 = new Chart(ctx2).Pie(pie2,{
+        barShowStroke: false,
+        scaleShowGridLines: false,
+        barValueSpacing : 20,
+    });
+    
+    var houseLabel = [
+    {% for house, score, color in categoryData.0.1 %}
+        "{{house}}"{% if not forloop.last %},{% endif %}
+    {% endfor %}
+    ]
+
+    {% for category, houses in categoryData %}
+     var {{category}}Data = {
+        labels: houseLabel,
+        datasets: [
+            {
+                label: "{{category}}",
+                fillColor: "rgba(217, 95, 73, 0.8)",
+                strokeColor: "rgba(217, 95, 73, 0.8)",
+                highlightFill: "{{houses.0.2}}",
+                highlightStroke: "rgba(217, 95, 73, 1)",
+                data: [{{houses.0.1}}, {{houses.1.1}}]
```

Comment:
This will only work if there are exactly two houses. Pull the data out, into a for loop.